### PR TITLE
Added organization_id to events

### PR DESCRIPTION
**Pull request by mllocs**
Little big detail, Events are org scoped, heh.

### CODE REVIEW

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<ul>
+  <% @events.each do |e| %>
+    <li>
+      <span><%= e.id %></span>
+      <span><%= e.action %></span>
+      <span><%= e.updated_at %></span>
+      <span><%= e.created_at %></span>
+    </li>
+  <% end %>
+</ul>
```
Comment (mllocs) [inline app/views/events/index.html.erb]: This is hidden for now

File: app/models/event.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ class Event < ActiveRecord::Base
   belongs_to :post
   belongs_to :member
   belongs_to :transfer
+  belongs_to :organization
```
Comment (enricostano) [inline app/models/event.rb]: Is this needed to `Event.where(organization_id: 27)`?

File: app/services/persister/transfer_persister.rb [hunk 3]
```diff
@@ -1,9 +1,10 @@
 module Persister
   class TransferPersister
-    attr_accessor :transfer
+    attr_accessor :transfer, :organization
 
-    def initialize(transfer)
+    def initialize(transfer, organization)
       @transfer = transfer
+      @organization = organization
```
Comment (enricostano) [inline app/services/persister/transfer_persister.rb]: Can't we infer from `@transfer.organization`?

File: app/models/event.rb (hunk 2 above)
Comment (mllocs) [reply app/models/event.rb]: nope, it's needed for `event.organization` and the setters (event.organization = organization), at least.

File: app/services/persister/transfer_persister.rb (hunk 3 above)
Comment (mllocs) [reply app/services/persister/transfer_persister.rb]: transfer doesn't have org id, It should be something like transfer.post.organization but then I realised that it was easier to pass it as an option to the Persister.

File: app/models/event.rb (hunk 2 above)
Comment (enricostano) [reply app/models/event.rb]: Yeah... I meant not literally the field, but 
the duplicated info in the column. if you don't need that kind of `where` you can infer the org from the associated resource

File: app/services/persister/transfer_persister.rb (hunk 3 above)
Comment (enricostano) [reply app/services/persister/transfer_persister.rb]: The less we expose the better, I vote for using `transfer.post.organization`

File: app/models/event.rb (hunk 2 above)
Comment (sauloperez) [reply app/models/event.rb]: Yep, I totally share the same view. Here we're duplicating information that will surely bite us in the shape of data inconsistency.

Comment (mllocs) [reply app/models/event.rb]: well, in theory ™️ events are read only, an not likely to be moved between orgs.

**Pull request closed without merging**